src/foundry/client-esm/applications/sheets/ambient-sound-config.d.mts

```diff
@@ -15,4 +17,31 @@ export default class AmbientSoundConfig<
   RenderContext,
   Configuration,
   RenderOptions
-> {}
+> {
+  static override DEFAULT_OPTIONS: DeepPartial<DocumentSheetV2.Configuration<AmbientSoundDocument.ConfiguredInstance>>;
+
+  static override PARTS: Record<string, HandlebarsApplicationMixin.HandlebarsTemplatePart>;
+
+  get title(): string;
+
+  protected override _prepareContext(options: DeepPartial<RenderOptions>): Promise<RenderContext>;
+
+  protected override _onRender(context: DeepPartial<RenderContext>, options: DeepPartial<RenderOptions>): Promise<void>;
+
+  protected override _onClose(options: DeepPartial<RenderOptions>): void;
+
+  override _onChangeForm(formConfig: ApplicationV2.FormConfiguration, event: Event): void;
+}
+
+declare namespace AmbientSoundConfig {
+  interface RenderContext {
+    sound: AmbientSoundDocument.ConfiguredInstance;
+    source: foundry.documents.BaseAmbientSound.Source;
+    fields: AmbientSoundDocument["schema"]["fields"];
```

Comment:
is `AmbientSoundDocument.Schema` not available?

Reply:
I think would be from the base document. Probably should also tackle that aliasing ticket.

Reply:
Yeah it'd be nice.